### PR TITLE
Fixed a few issues reported, and added gunpowder to Creepers 

### PR DESCRIPTION
**Pull request by Xwaffle1**
Fixed Sheep wool color on drop. (Fix Issue #1573)

Added gunpowder drop to Creeper

Fix Issue #1698 

Fix Issue #1497

### CODE REVIEW

**Comment by Xwaffle1** [comment]
Pull Requests take a while to get accepted?

**Comment by Tee7even** [comment]
>Pull Requests take a while to get accepted?

Guessed it.

**Comment by Xwaffle1** [comment]
Lol. Why is that? Do multiple people have the permission to merge pull requests?

**Comment by PikyCZ** [comment]
tee7even can merge this pull ..

File: src/main/java/cn/nukkit/entity/passive/EntitySheep.java [hunk 1]
```diff
@@ -117,9 +117,9 @@ public boolean shear() {
     }
 
     public void setColor(int color) {
-        this.namedTag.putByte("Color", this.color);
         this.color = color;
-        this.setDataProperty(new ByteEntityData(DATA_COLOUR, color));
+		this.namedTag.putByte("Color", this.color);
```
Comment (Tee7even) [inline src/main/java/cn/nukkit/entity/passive/EntitySheep.java]: Indentation.

File: src/main/java/cn/nukkit/Player.java [hunk 2]
```diff
@@ -3386,6 +3386,8 @@ public void handleDataPacket(DataPacket packet) {
                         case Item.DIAMOND:
                             this.awardAchievement("diamond");
                             break;
+                            default:
```
Comment (Tee7even) [inline src/main/java/cn/nukkit/Player.java]: Indentation.

**Comment by Xwaffle1** [comment]
There we go, finished with requested changes. Thanks!

**Comment by Tee7even** [comment]
It doesn't exactly fix #1698, but just disables error output. But that's for a later time.

**Comment by Xwaffle1** [comment]
#1698 only occured because it's trying to give achievements for items that don't give out achievements, like when he crafted a torch it was going through the switch statement when there was no case for a torch

**Pull request merged**